resolwe/flow/tests/test_commands.py

```diff
@@ -141,3 +141,21 @@ def test_list_has_test_image_yaml(self):
         self.assertTrue(len(imgs) != 0)
         self.assertTrue(isinstance(imgs[0], dict))
         self.assertTrue(dict(name='resolwe/test', tag='base') in imgs)
+
+    def test_list_is_sorted(self):
+        # The returned list must be sorted alphabetically
+        out, err = StringIO(), StringIO()
+        call_command('list_docker_images', stdout=out, stderr=err)
+        self.assertEqual('', err.getvalue())
+        self.assertNotEqual('', out.getvalue())
+        lines = out.getvalue().split('\n')[:-1]
+        self.assertEqual(lines, sorted(lines))
+
+    def test_list_versions(self):
+        # The returned list must contain only the Docker image from
+        # the latest version of a given process
+        out, err = StringIO(), StringIO()
+        call_command('list_docker_images', stdout=out, stderr=err)
+        self.assertEqual('', err.getvalue())
+        self.assertTrue('resolwe/test:versioning-2' in out.getvalue())
```

Comment:
You can use [`assertIn()` and `assertNotIn()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIn) here.

Reply:
Fixed, thanks!